ui/src/dashboards/components/createFromTemplateOverlay/DashboardTemplateDetails.tsx

```diff
@@ -0,0 +1,149 @@
+        className="import-template-overlay--details"
+        autoSize={false}
+      >
+        <Panel
+          size={ComponentSize.Medium}
+          testID="template-panel"
+          className="import-template-overlay--panel"
+        >
+          <Panel.Body>{this.panelContents}</Panel.Body>
+        </Panel>
+      </DapperScrollbars>
+    )
+  }
+
+  private get panelContents(): JSX.Element {
+    const {selectedTemplateSummary} = this.props
+
+    if (!selectedTemplateSummary) {
+      return (
+        <EmptyState size={ComponentSize.Medium}>
+          <EmptyState.Text text="Select a Template from the left" />
+        </EmptyState>
+      )
+    }
+
+    return (
+      <Grid>
+        <Grid.Row>
+          <Grid.Column widthSM={Columns.Twelve}>
+            {this.templateName}
+            {this.templateDescription}
+          </Grid.Column>
+          <Grid.Column widthSM={Columns.Six}>
+            <h5 className="import-template-overlay--heading">Variables</h5>
+            {this.variablesList}
+          </Grid.Column>
+          <Grid.Column widthSM={Columns.Six}>
+            <h5 className="import-template-overlay--heading">Cells</h5>
+            {this.cellsList}
+          </Grid.Column>
+        </Grid.Row>
+      </Grid>
+    )
+  }
+
+  private get variablesList(): JSX.Element | JSX.Element[] {
+    const {variables} = this.props
+
+    if (!variables.length) {
+      return (
+        <p className="import-template-overlay--included missing">
+          No included variables
+        </p>
+      )
+    }
+
+    return variables.map((variable, i) => (
+      <p
+        className="import-templates-overlay--included"
+        key={`${i} ${variable}`}
+      >
+        {variable}
+      </p>
+    ))
+  }
+
+  private get cellsList(): JSX.Element | JSX.Element[] {
+    const {cells} = this.props
+
+    if (!cells.length) {
+      return (
+        <p className="import-template-overlay--included missing">
+          No included cells
+        </p>
+      )
+    }
+
+    return cells.map((cell, i) => (
+      <p className="import-templates-overlay--included" key={`${i} ${cell}`}>
+        {cell}
+      </p>
+    ))
+  }
+
+  private get templateDescription(): JSX.Element {
+    const {selectedTemplateSummary} = this.props
+    const description = _.get(selectedTemplateSummary, 'meta.description')
+
+    if (description) {
+      return (
+        <p className="import-template-overlay--description">{description}</p>
+      )
+    }
+
+    return (
+      <p className="import-template-overlay--description missing">
+        No description
+      </p>
+    )
+  }
+
+  private get templateName(): JSX.Element {
+    const {selectedTemplateSummary} = this.props
+    const name = _.get(selectedTemplateSummary, 'meta.name')
+    const version = _.get(selectedTemplateSummary, 'meta.version')
+
+    let templateName = name || 'Untitled'
+
+    if (version) {
+      templateName = `${name || 'Untitled'} (v${version})`
```

Comment:
hrmm. we are not currently using version in a way that makes it make sense to expose it so saliently to the user. I'm actually a little unsure how we should use versions. Maybe let's not display this to user. ?

Reply:
Sure! I figure it will be more useful in the future